pulsar-common/src/main/java/org/apache/pulsar/common/functions/FunctionConfig.java

```diff
@@ -87,6 +87,7 @@
     private String logTopic;
     private ProcessingGuarantees processingGuarantees;
     private Boolean retainOrdering;
+    private Boolean retainKeyOrdering;
```

Comment:
Can we add some comments here for retainOrdering and retainKeyOrdering so people understand what they do and what the difference between the two is?

Reply:
also need to add this for SinkConfig